### DISCUSSION TITLE
Vite Deploy Css not loading since yesterday

### DISCUSSION THREAD

**Discussion by stableneo** [community/community#69414] [Programming Help] [answered]
### Body

Hello GitHub Community,

I am relatively new to GitHub Pages and have been working on my website without any issues until yesterday.

However, today I encountered a strange problem where my website no longer loads its CSS, and I'm not sure what caused this issue. I haven't made any significant changes to the repository, and it was working perfectly before.

I've checked the paths and file names, and they appear to be correct. I also tried clearing my browser cache and accessing the website from different browsers, but the issue persists.

This is the repository: https://github.com/stableneo/website.

I followed this tutorial on youtube to setup my website on github: https://youtu.be/XhoWXhyuW_I.

I would greatly appreciate any insights or guidance on how to resolve this problem. If anyone has encountered a similar issue or knows of potential solutions, please let me know. Your assistance would be invaluable to me as I continue to learn and grow on GitHub.

Thank you in advance for your help and support.

Neo

### Guidelines

- [x] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebubekirgungor** [comment, marked as answer]
Hello, in https://github.com/stableneo/website/blob/main/vite.config.js file you should set the base as "website" like:
`export default defineConfig({
  plugins: [react()],
  base: "/website"
})`

**Reply by stableneo** [reply]
It worked thank you! Could you tell me why tho? In the Vite Docs it says this:

If you are deploying to https://<USERNAME>.github.io/<REPO>/, for example your repository is at https://github.com/<USERNAME>/<REPO>, then set base to '/<REPO>/'.

**Reply by ebubekirgungor** [reply]
You are welcome. Your website url is https://stableneo.github.io/website/ but in config file it was "/vite-deploy". It must be the same as your repository name.

**Reply by stableneo** [reply]
Yeah i got it now. I changed the repository name 3-4 days ago and it still worked so i forgot to change it in the file... Thank you so much for the fast answer!

**Reply by ebubekirgungor** [reply]
You are welcome

**Reply by Shagogcst** [reply]
Laptop Specification
Model Elitebook 840 G5
Processor Core i5
Generation 8th Generation
RAM 8GB DDR4
Storage 256GB SSD
This issue is 64bit 32bit.
OS :ubuntu-22-4
Multi2sim-5.0
How to solve this issue?
Issue: [X86] error: unspported instruction (f3 0f 1e fb)
Best regards
